File: 프로그래머스/2/42746. 가장 큰 수/가장 큰 수.py
def solution(numbers):
    answer = ''
    numbers = list(map(str,numbers))
    
    if all(num == '0' for num in numbers) :
        return '0'
    
    numbers.sort(key=lambda x:x*3, reverse=True)
    answer = ''.join(numbers)
    
    return answer

File: 프로그래머스/2/42746. 가장 큰 수/가장 큰 수.js
function solution(numbers) {
    var answer = '';
    numbers.sort((a, b) => {
        let strA = a.toString();
        let strB = b.toString();
        return (strB + strA) - (strA + strB);
    });
    
    if (numbers[0]===0) return "0";
    answer = numbers.join('');
    
    return answer;
}